QaranyoMadhaneAlam/QM001.xml

```diff
@@ -0,0 +1,178 @@
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0921QM"/>
+                        <idno>QM001</idno>
+                      <altIdentifier>
+                         <idno>Inventory number: AW 003</idno>
+                      </altIdentifier>
+                        <altIdentifier>
+                      <idno>Inventory number:G1-IV-326</idno>
+                   </altIdentifier>
+                   <altIdentifier>
+                 <idno>Inventory number:HE-IV-1</idno>
+              </altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                       <summary/>
+                           <msItem xml:id="ms_i1">
+                          <locus from="3r" to="96v"/>
+                          <title type="complete" ref="LIT1693John"/>
+
+                          <msItem xml:id="ms_i1.1">
+                          <locus target="#3r"/>
+                          <title type="complete" ref="LIT1693John#JohnGospelHeadline"><hi rend="rubric"> ብስራተ፡ አብ፡ ትሩፍ፡ ረድእ፡ ተናጋሪ፡ በመለኮት፡
+                            ዮሐንስ፡ ወልደ፡ዘብዴዎስ፡፡ ፍቁረ፡ እግ</hi>ዚአብሔር
+                            </title>
+                            </msItem>
+                              <msItem xml:id="ms_i1.2">
+                                <locus from="3r" to="89va"/>
+                                <title type="complete" ref="LIT1693John#JohnGospel"/>
+                                   <incipit><hi rend="rubric">ቀዳሚሁ፡ ውእቱ፡ ቃል፡ ወውእቱ፡ ቃል፡ ኀ</hi>በ፡ እግዚኣብሔር፡ ሀለወ፡ ወእግዚኣብሔር፡ ውእቱ፡
+                                    ቃል፡ ዝንቱ፡ እምቀዲሙ፡ ሀሎ፡ ኀበ፡ እግዚኣብሔር። ኵሉ፡ ቦቱ፡ ኮነ፡ ወዘእንበሌሁሰ፡ አልቦ፡ ዘኮነ፡፡</incipit>
+                                    <explicit>ወሶበ፡ ተጽሕፈ፡ ኵሉ፡ በበ፡ ኣሓዱ፡ እምኢያገምሮ፡ዓለም፡ ጥቀ፡ መጻሕፍቲሁ፡ ዘእምተጽሕፈ፡ አሜን። ወአሜን። ለይኩን ለይኩን። ወ</explicit>
+                                  </msItem>
+                            <msItem xml:id="ms_i1.3">
+                              <locus target="#89v"/>
+                              <title type="complete" ref="LIT1693John#Postscript"/>
+                              <incipit>መልዓ፡ስብከተ፡ ዮሐንስ</incipit>
+                              <explicit>ወበ፷፡ ዓመተ፡ መንግሥቱ፡ </explicit>
+                                <explicit type="subscriptio">ዝ መጽሐፍ ዘ <persName ref="PRS13313Labbawekrestos" role="owner">ለባዊ ክርስቶስ</persName> ጸሐፊሁ <persName ref="PRS13314Haylamasqal" role="scribe">ኃይለ መስቀል</persName> ወአቡሁ <persName ref="PRS13315TasfaIyyasus"> ተስፋ ኢየሱስ</persName> ወእሙ <persName ref="PRS13316WalattaDawit">ወለተ ዳዊት</persName> ወብእሲቱ <persName ref="PRS13328AbalaMaryam">አባለ ማርያም</persName> ወውልዱ <persName ref="PRS13329Pestqis">ጵስጥቂስ</persName> ዘሠረቆ ወዘፈሐቆ</explicit>
```

Comment:
I would say that this is not a subscriptio, but I could not find the way of encoding this in the GL.